src/pages/Signup.tsx

```
@@ -0,0 +1,230 @@
+];
+
+const Signup = () => {
+  const navigate = useNavigate();
+  const [mail, setMail] = useState("");
+  const [password, setPassword] = useState("");
+  const [checkPassword, setCheckPassword] = useState("");
+  const [interest, setInterest] = useState<Interest[]>([]);
+  const [showPassword, setShowPassword] = useState(false);
+  const [showCheckPassword, setShowCheckPassword] = useState(false);
+  const { isLogin } = useContext(AuthContext);
+  const [errors, setErrors] = useState<Record<string, string>>({
```

ESLint

[error] 'errors' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)